### PR TITLE
Refactor DraftsPage/List terms and some code cleanup

### PR DESCRIPTION
**Pull request by darkruby501**
┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1202943435983100) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/editor/EditorFormComponent.tsx [hunk 1]
```diff
@@ -122,9 +122,9 @@ export const EditorFormComponent = ({form, formType, formProps, document, name,
   
   const onRestoreLocalStorage = useCallback((newState: EditorContents) => {
     wrappedSetContents({contents: newState, autosave: false});
-    if (editorRef.current)
-      editorRef.current.focusOnEditor();
-  }, [editorRef, wrappedSetContents]);
+    // if (editorRef.current)
+      // editorRef.current.focusOnEditor(); //function not currently implemented
```
Comment (jpaddison3) [inline packages/lesswrong/components/editor/EditorFormComponent.tsx:126]:
```suggestion
    // TODO: Focus editor
```

File: packages/lesswrong/components/posts/PostsItem2.tsx [hunk 2]
```diff
@@ -365,16 +343,31 @@ const PostsItem2 = ({
...
+  /* relevance to that tag.*/
   tagRel?: WithVoteTagRel|null,
+  /** defaultToShowComments: (bool) If set, comments will be expanded by default.*/
   defaultToShowComments?: boolean,
+  /** sequenceId, chapter: If set, these will be used for making a nicer URL.*/
   sequenceId?: string,
   chapter?: any,
+  /** index: If this is part of a list of PostsItems, its index (starting from
+   /* zero) into that list. Used for special casing some styling at start of
+   /* the list.*/
```
Comment (jpaddison3) [inline packages/lesswrong/components/posts/PostsItem2.tsx:359]:
```suggestion
  /* zero) into that list. Used for special casing some styling at start of
  /* the list.*/
```

File: packages/lesswrong/components/posts/PostsItem2.tsx [hunk 3]
```diff
@@ -365,16 +343,31 @@ const PostsItem2 = ({
...
   index?: number,
+  /** terms: If this is part of a list generated from a query, the terms of that
+  /* query. Used for figuring out which sticky icons to apply, if any.*/
```
Comment (jpaddison3) [inline packages/lesswrong/components/posts/PostsItem2.tsx:362]:
```suggestion
  /**
   * terms: If this is part of a list generated from a query, the terms of that
   * query. Used for figuring out which sticky icons to apply, if any.
   */
```

Jim would like to be on on record that this is the idiomatic style for the codespace.

Comment (jpaddison3) [reply packages/lesswrong/components/posts/PostsItem2.tsx:362]: But no need to change them all

**Review by jpaddison3 — APPROVED** [review]
LGTM

File: packages/lesswrong/components/posts/PostsItem2.tsx (hunk 3 above)
Comment (darkruby501) [reply packages/lesswrong/components/posts/PostsItem2.tsx:362]: Noted!

**Pull request merged**